extensions/ql-vscode/src/vscode-tests/cli-integration/query.test.ts

```diff
@@ -81,65 +83,36 @@ const queryTestCases: QueryTestCase[] = [
 
 describe('using the query server', function() {
   before(function() {
-    if (process.env['CODEQL_PATH'] === undefined) {
-      console.log('The environment variable CODEQL_PATH is not set. The query server tests, which require the CodeQL CLI, will be skipped.');
+    if (!hasCodeQL()) {
```

Comment:
I agree it would be nice to separate the tests that need the standard libraries from those that don't, but I also agree this can be done later.

Reply:
Actually I think this particular set of tests doesn't need the standard libraries.

Reply:
Because this test now calls the cli in the same way as the rest of the extension, there is a qlpack in scope and the standard libraries are required.

I'm not sure if it's possible to run a query without a qlpack in scope inside of the extension.

Reply:
These particular tests do not use the standard libraries though: there is no import statement. You do need a sample qlpack.yml file next to the test queries (I think we have one).
Happy to merge and see if we can weaken the requirement later.

Reply:
Right...the qlpack being used here explicitly requires the javascript libs. We would need to use a different qlpack that doesn't require any dependent libraries and to do that, we'd need to put them in a different directory since this directory is shared with "real" queries that actually query a database.

Reply:
Ohh yes this makes sense now.
- https://github.com/github/vscode-codeql/tree/main/extensions/ql-vscode/test/data is missing a `qlpack.yml`, and should have one that points to `test.dbscheme` and no other dependencies.
- https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/src/vscode-tests/cli-integration/data is missing a dbscheme file, and its `qlpack.yml` should point to that dbscheme file and no other dependencies.

Reply:
OK. Let me see if I can get that working. Also, not a fan that we have some tests in `ql-vscode/test` and others in `ql-vs-code/src/vscode-tests`.